playbook-control-room-desktops.yml

```diff
@@ -1,5 +1,6 @@
 ---
 - hosts: control_room:linac_opi:fac_control_room:fac_servers
+# - hosts: tmp_group
```

Comment:
Isn't it better to include the host group in the hosts list above? Or we do not want to update the webserver when running the playbook?

Reply:
Actually, doesn't it make sense to use a different playbook for the host? serv_web is very different from the other "desktops" and it would ease understanding

Reply:
comment line for easy testing.